pkg/handler/triggerhandler.go

```diff
@@ -532,7 +532,7 @@ func request_PipelinePublicService_TriggerAsyncNamespacePipeline_0(ctx context.C
 }
 
 // ref: the generated protogen-go files
-func request_PipelinePublicService_TriggerAsyncNamespacePipeline_0_form(ctx context.Context, marshaler runtime.Marshaler, client pb.PipelinePublicServiceClient, req *http.Request, pathParams map[string]string) (proto.Message, runtime.ServerMetadata, error) {
```

Comment:
This is copied from the generated code in `protogen-go` but it doesn't seem to override the methods defined there (they're package methods that don't belong to any type) so this rename should be fine 👍 cc @donch1989